apps/builder/app/builder/builder.tsx

```diff
@@ -294,6 +298,11 @@ export const Builder = ({
   useSetWindowTitle();
 
   const iframeRefCallback = mergeRefs((element: HTMLIFrameElement | null) => {
+    if (element?.contentWindow) {
+      // added to iframe window and stored in local variable right away to prevent
+      // overriding in emebedded scripts on canvas
+      element.contentWindow.syncEmitter = builderClient.emitter;
```

Comment:
you are adding this syncEmitter variable on the window/global ... that should be at least some lengthy name aka `__webstudioSomething__`

Reply:
Doesn't really matter, can rename